src/uu/chroot/src/chroot.rs

```diff
@@ -63,7 +63,7 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     }
 
     let commands = match matches.get_many::<String>(options::COMMAND) {
-        Some(v) => v.map(|s| s.as_str()).collect(),
+        Some(v) => v.map(std::string::String::as_str).collect(),
```

Comment:
i would prefer just
String::as_str
and have an import

repeating std::string isn't providing much value